pkg/cli/create_helm.go

```diff
@@ -247,30 +278,6 @@ func CreateHelm(ctx context.Context, options *CreateOptions, globalFlags *flags.
 		return err
 	}
 
-	// check if vcluster already exists
-	if !cmd.Upgrade {
```

Comment:
Why is that removed?

Reply:
It looked like a duplicate of line 133-151.